apps/src/templates/HintsDisplay.jsx

```diff
@@ -0,0 +1,60 @@
+module.exports = React.createClass({
+
+  propTypes: {
+    hintReviewTitle: React.PropTypes.string.isRequired,
+    seenHints: React.PropTypes.array.isRequired,
+    unseenHints: React.PropTypes.array.isRequired,
+    onUserViewedHint: React.PropTypes.func.isRequired,
+    lightbulbSVG: React.PropTypes.node.isRequired,
+  },
+
+  getInitialState: function () {
+    return {
+      showNextUnseenHint: false
+    };
+  },
+
+  viewHint: function () {
+    this.props.onUserViewedHint();
+    this.setState({
+      showNextUnseenHint: true
+    });
+  },
+
+  render: function () {
+
```

Comment:
I notice you often have this blank first link in a lot of your functions. Any argument for it? I don't necessarily have a problem with it.